metriq_gym/qplatform/device.py

```diff
@@ -2,17 +2,17 @@
 from typing import cast
 
 import networkx as nx
+import rustworkx as rx
+import qnexus as qnx
 from qbraid import QuantumDevice
 from qbraid.runtime import AzureQuantumDevice, BraketDevice, QiskitBackend
 from qiskit.transpiler import CouplingMap
 
-
-import rustworkx as rx
-
 from metriq_gym.local.device import LocalAerDevice
+from metriq_gym.quantinuum.device import QuantinuumDevice
 
 
-### Version of a device backend (e.g. ibm_sherbrooke --> '1.6.73') ###
+# Version of a device backend (e.g. ibm_sherbrooke --> '1.6.73').
 @singledispatch
 def version(device: QuantumDevice) -> str:
```

Comment:
we have now found a way to get the version of Quantinuum device, we may as well use it and implement it here